commands/station.js

```diff
@@ -103,14 +101,7 @@ export const station = async ({ json, experimental }) => {
   ]
 
   if (experimental) {
-    modules.push(pRetry(() => bacalhau.run({
-      FIL_WALLET_ADDRESS,
-      storagePath: join(paths.moduleCache, 'bacalhau'),
-      onActivity: activity => {
-        activities.submit({ source: 'Bacalhau', ...activity })
-      },
-      onMetrics: m => metrics.submit('bacalhau', m)
-    }), { retries: 1000 }))
+    // No experimental module available at this point
```

Comment:
Should we print a message to let the user know their CLI arg is being ignored?

Reply:
great detail!